README.md

```diff
@@ -1 +1,13 @@
-# hw01
\ No newline at end of file
+# hw01
+## Introduction
+Hi, my name is Roy, and I am a visiting undergraduate student at the University of Chicago! I study *Economic History and Economics* at the **London School of Economics**, and I am here to learn some basic programming that will help me with my major.
+
+**List of things about me:**
+* I am from Singapore
+  * Tiny island in Southeast Asia, barely larger than Chicago city!
+* I am a big soccer fan
+  * Team I support is [Arsenal FC](https://www.arsenal.com/)
+* Music taste: R&B, Rap, some Pop
+
+That's all! Have a picture of the Singapore skyline because why not.
+![Singapore skyline](https://media.istockphoto.com/photos/singapore-skyline-at-marina-bay-at-twilight-with-glowing-sunset-the-picture-id642250848?k=6&m=642250848&s=612x612&w=0&h=j6axpCSSN2_j8OOjuvrs0qzo1FcKv2wvlKOlsV_rG4g=)
```

Comment:
Great photo! FYI, for some reason, GitHub won’t show the captions you add to images/tables. I am not sure why this is the case with GitHub because the captions _do_ show up in other formats (like RStudio previews).